app/config/config_test.yml

```diff
@@ -51,5 +51,6 @@ sylius_theme:
     sources:
         test: ~
 
-twig:
-    exception_controller: "Sylius\\Bundle\\CoreBundle\\Controller\\VerboseExceptionController::showAction"
+fos_rest:
+    exception:
+        debug: true
```

Comment:
If I am correct, debug is true for env ``test`` and default fos_rest config takes it from the kernel's debug setting. We should add this to ``config_test_cached``, which has disabled the debug mode, thus creating the need for VerboseExceptionController in the past. WDYT?